### ISSUE TITLE
resource/aws_route: Add LocalGatewayId attribute support

### ISSUE THREAD

**Issue by FireballDWF** [terraform-providers/terraform-provider-aws#13888]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Add ability to define a route via the LGW

### New or Affected Resource(s)

* aws_route

### Potential Terraform Configuration

```hcl
local_gateway_id  - (Optional) The ID of the local gateway 
```

### References

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateRoute.html#API_CreateRoute_RequestParameters

* #0000

**Comment by ewbankkit** [comment]
Also affected:

* [`aws_route_table`](https://www.terraform.io/docs/providers/aws/r/route_table.html)
* [`aws_default_route_table`](https://www.terraform.io/docs/providers/aws/r/default_route_table.html)

and associated data sources.

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 3.5.0 of the Terraform AWS Provider, likely on Thursday. Thanks to @johnbarney for the implementation and @FireballDWF for the testing assistance. 👍
